runtime-sdk/src/modules/accounts/mod.rs

```diff
@@ -75,12 +81,15 @@ pub enum Event {
 pub struct Parameters {
     #[serde(rename = "transfers_disabled")]
     pub transfers_disabled: bool,
+    #[serde(rename = "gas_cost_tx_transfer")]
+    pub gas_cost_tx_transfer: u64,
```

Comment:
Maybe a separate "GasCosts" struct and then have "tx_transfer" as field?

Reply:
:shrug: okay